packages/forms/src/model.ts

```diff
@@ -168,8 +168,7 @@ function isOptionsObj(validatorOrOpts?: ValidatorFn|ValidatorFn[]|AbstractContro
  */
 export abstract class AbstractControl {
   /** @internal */
-  // TODO(issue/24571): remove '!'.
-  _pendingDirty!: boolean;
+  _pendingDirty: boolean = false;
```

Comment:
Since this change (and similar changes below) adds some extra bytes, I'm curious if there is an additional refactoring that you plan to propose (since you've mentioned "This eliminates the need to constantly check for truthiness and makes code much more readable." in the commit message), which would reduce the size based on this change?

Reply:
This is a bit more readable, seems worth a few bytes to me

Reply:
Right now, I don't have any propose about refactoring plan, I'll do whatever you are suggesting🙂

Reply:
I've looked at the code and I don't think there are other related changes that we can make. However as @dylhunn mentioned, that improves code readability, so we can keep this. In this case we can drop the `boolean` type annotation - the `boolean` would be inferred by TypeScript.

```suggestion
  _pendingDirty = false;
```